horace_core/sqw/@sqw/generate_cube_sqw.m

```diff
@@ -0,0 +1,65 @@
+function out = generate_cube_sqw(shape)
+    % Generate a cube of SQW data FOR TESTING CUT/SYMMETRISE
+    % The data will be invalid in most circumstances
```

Comment:
what does invalid mean in this context?

Reply:
It means that you can't actually use the data for much beyond cuy/symmetrise and will generally result in crashes or nonsense if used outside of those.